src/open_dread_rando/specific_patches/static_fixes.py

```diff
@@ -342,6 +342,36 @@ def apply_experiment_fixes(editor: PatcherEditor):
         "actor": "trap_thermal_horizontal_POSTCOOL"
     }, "mapDoors")
 
+    # add thermal door in front of morph launcher
+    new_door = copy.deepcopy(editor.resolve_actor_reference({
+        "scenario": "s020_magma",
+        "actor": "trap_thermal_horizontal_004"
+    }))
+
+    new_name = "trap_thermal_horizontal_006"
+
+    new_door.sName = new_name
+    new_door.vPos = [5840.0, -5455.0, 0.0]
+
+    magma.actors_for_layer('default')[new_name] = new_door
+    for group in ["eg_collision_camera_004_Default", "eg_collision_camera_004_PostXRelease"]:
```

Comment:
ideally iterate through all the actor groups the original trapdoor is in, just to be safe